lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -34,20 +34,20 @@ class KubernetesMetadataFilter < Fluent::Filter
                  :default => '\.(?<pod_name>[^\._]+)_(?<namespace>[^_]+)_(?<container_name>.+)-(?<docker_id>[a-z0-9]{64})\.log$'
     config_param :bearer_token_file, :string, default: ''
     config_param :merge_json_log, :bool, default: true
+    config_param :include_namespace_id, :bool, default: false
 
-    def get_metadata(namespace, pod_name, container_name)
+    def get_metadata(namespace_name, pod_name, container_name)
       begin
-        metadata = @client.get_pod(pod_name, namespace)
-        if metadata
-          return {
-              uid:            metadata['metadata']['uid'],
-              namespace:      metadata['metadata']['namespace'],
-              pod_name:       metadata['metadata']['name'],
-              container_name: container_name,
-              labels:         metadata['metadata']['labels'].to_h,
-              host:           metadata['spec']['host']
-          }
-        end
+        metadata = @client.get_pod(pod_name, namespace_name)
+        return if !metadata
+        return {
+            namespace_name: namespace_name,
```

Comment:
Regardless of the id issue, `namespace_name` closely parallels both `pod_name` and `container_name`.

As mentioned, though, I'm perfectly happy to name them whatever you'd like, as out_google_cloud can remap them without any issue, so just let me know what you'd like for both `namespace` and `uid`.